src/js/rest/postDeploy.js

```diff
@@ -0,0 +1,32 @@
+import createError from './common/createError';
```

Comment:
I think the file name should be more clear
+ postDeploy => postDeployAppSettings